lib/iris/tests/test_merge.py

```diff
@@ -187,7 +190,7 @@ def test__ndarray_ndarray(self):
             expected = self._make_data([0, 1], dtype=self.dtype)
             self.assertArrayEqual(result.data, expected)
             self.assertEqual(result.dtype, self.dtype)
-            self._check_fill_value(result, fill0, fill1)
+            self._check_fill_value(result)
```

Comment:
For tests including ndarray (e.g. `test__ndarray_ndarray`, `test__masked_ndarray`, `test__ndarray_masked`) it seems unnesccary to test the `fill_combos` since we already know they won't have fill_values